NAMESPACE

```diff
@@ -26,6 +26,7 @@ export(erddap_grid)
 export(erddap_info)
 export(erddap_search)
 export(erddap_table)
+export(gefs)
```

Comment:
Looks like you used `@export` tags in the `gefs.R` file, but maybe you need to re-document/roxygenize as those exported functions should be in the NAMESPACE file